packages/roosterjs-editor-core/lib/editor/adjustBrowserBehavior.ts

```diff
@@ -0,0 +1,38 @@
+import { Browser } from 'roosterjs-editor-dom';
+import { DocumentCommand } from 'roosterjs-editor-types';
+
+const COMMANDS: {
+    [command: string]: any;
+} = Browser.isFirefox
+    ? {
+          /**
+           * Disable these object resizing for firefox since other browsers don't have these behaviors
+           */
+          [DocumentCommand.EnableObjectResizing]: false,
+          [DocumentCommand.EnableInlineTableEditing]: false,
+      }
+    : Browser.isIE
+    ? {
+          /**
+           * Change the default paragraph separater to DIV. This is mainly for IE since its default setting is P

```

Comment:
Should this only be done in IE? The comment implies it can be done in other browsers as well

Reply:
hmm, the comment was moved from editor.ts. It means this is only for IE.
Since this is just a code integration from master, I prefer change it. We can refine it later in master branch.